app.py

```diff
@@ -50,7 +51,21 @@ def _get_local_time():
             return f"Could not process time information. {e}", 404
 
 
-#TODO Add a route that displays the current time for all US timezones.
+@app.route('/time-difference', methods=['GET', 'POST','PUT'])
+def time_diff():
+    try:
+        if request.args:
+            h = request.args.get('here')
```

Comment:
I think that 'here' and 'there' are a little confusing, I suggest maybe 'timezone1' and 'timezone2'